README.md

```diff
@@ -110,13 +110,24 @@ packages were manually upgraded.
 
 ## Filing Issues
 
-This repo does not accept issues. Please file issues in 
+This repo does not accept issues. Please file issues in
 [dotnet/source-build](https://github.com/dotnet/source-build/issues/new/choose).
 
 ## Cleanup
```

Comment:
What is the guidance for updating the SBRP self-reference? Should that be described here? Is it something that should be done for each release (in which case it should be part of the release checklist)?

Reply:
That should be considered an optional release/rebootstrap activity.  It shouldn't be updated to anything other than the version included in the previous artifacts.

Reply:
I opened https://github.com/dotnet/release/pull/1011 for this.